diff -Naur ulogd-2.0.8-orig/ulogd.conf.in ulogd-2.0.8/ulogd.conf.in
--- ulogd-2.0.8-orig/ulogd.conf.in	2022-11-02 11:12:52.296690448 +0200
+++ ulogd-2.0.8/ulogd.conf.in	2023-07-02 11:18:30.349569740 +0300
@@ -8,7 +8,7 @@
 
 
 # logfile for status messages
-logfile="/var/log/ulogd.log"
+logfile="/var/log/ulogd/ulogd.log"
 
 # loglevel: debug(1), info(3), notice(5), error(7) or fatal(8) (default 5)
 # loglevel=1
@@ -184,15 +184,15 @@
 socket_path="/tmp/nuauth_ulogd2.sock"
 
 [emu1]
-file="/var/log/ulogd_syslogemu.log"
+file="/var/log/ulogd/syslogemu.log"
 sync=1
 
 [op1]
-file="/var/log/ulogd_oprint.log"
+file="/var/log/ulogd/oprint.log"
 sync=1
 
 [gp1]
-file="/var/log/ulogd_gprint.log"
+file="/var/log/ulogd/gprint.log"
 sync=1
 timestamp=1
 
@@ -323,7 +323,7 @@
 
 [nacct1]
 sync = 1
-#file = /var/log/ulogd_nacct.log
+#file = /var/log/ulogd/nacct.log
 
 [mark1]
 mark = 1
